src/lib-storage/mail-search-register-human.c

```diff
@@ -172,6 +172,11 @@ static const struct mail_search_register_arg human_register_args[] = {
 	{ "LARGER", human_search_larger },
 	{ "SMALLER", human_search_smaller },
 
+	/* OBJECTID extension: */
+	{ "EMAILID", human_search_guid },
+	{ "THREADID", human_search_guid },
+	{ "MAILBOXID", human_search_mailbox_guid },
```

Comment:
Also no MAILBOXID for SEARCH.

Reply:
same as MAILBOXID for FETCH, I am planning to use it in our webmail client